subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/types.go

```diff
@@ -37,11 +37,12 @@ type (
 )
 
 type schedulingRequest struct {
-	vc            api.VirtualClusterName
-	reservationId api.ReservationId
-	chain         CellChain
-	affinityGroup map[int32]int32 // gpu number -> pod number
-	priority      CellPriority
+	vc                api.VirtualClusterName
+	reservationId     api.ReservationId
+	chain             CellChain
+	affinityGroupName string
+	affinityGroup     map[int32]int32 // gpu number -> pod number
```

Comment:
is it still suitable to name the fifth member as "affinityGroup" when there is an "affinityGroupName"? Looks a bit confusing. How about "affinityGroupGpuNums" or something like that?

Reply:
Renamed to affinityGroupPodNums according to totalPodNums in AlgoAffinityGroup

---
In reply to: [351584723](https://github.com/microsoft/pai/pull/3917#discussion_r351584723) [](ancestors = 351584723)